### PR TITLE
feat(ui5 application writer): trim themelib configuration property

### PR DESCRIPTION
**Pull request by korotkovao**
Trim `themelib` property value.
https://github.com/SAP/open-ux-tools/issues/590

### CODE REVIEW

**Review by devinea — COMMENTED** [review]
Changes and test look good. Thanks @korotkovao 
Can you add a match changeset?
https://github.com/SAP/open-ux-tools/blob/9d4e3bcfd4a1d45e93ba603ecf345e34896cedf8/README.md#create-changesets-for-feature-or-bug-fix-branches

File: packages/ui5-config/src/ui5config.ts [hunk 1]
```diff
@@ -91,7 +91,7 @@ export class UI5Config {
             libraryObjs.push({ name: library });
         }
         // Add theme lib (dark theme versions are provided by base theme lib)
-        libraryObjs.push({ name: `themelib_${ui5Theme.replace(/_dark$/, '')}` });
+        libraryObjs.push({ name: `themelib_${ui5Theme.replace(/_dark|_hcw|_hcb/, '')}` });
```
Comment (IainSAP) [inline packages/ui5-config/src/ui5config.ts:94]: Are you sure about dropping the `$`. This changes the regex match to match anywhere in the string not just the end. I think we should retain the precision.

**Review by IainSAP — CHANGES_REQUESTED** [review]

File: packages/ui5-config/src/ui5config.ts (hunk 1 above)
Comment (korotkovao) [reply packages/ui5-config/src/ui5config.ts]: Thanks @IainSAP, i have added it back

**Review by IainSAP — DISMISSED** [review]
Thx @korotkovao

**Review by devinea — DISMISSED** [review]
Thanks @korotkovao

**Comment by korotkovao** [comment]
Thank you @devinea and @IainSAP

**Review by IainSAP — APPROVED** [review]

**Pull request merged**